tests/test_admin.py

```diff
@@ -48,6 +48,18 @@ def test_changelist_not_accessible_to_regular_users(self):
             response, "/en/admin/login/?next=/en/admin/cms/pagecontent/"
         )
 
+    def test_ordering_author(self):
+        model = PageContent
+        order = 2
+        author = PageContentAdmin._list_display[order]
+        self.assertEqual(author, 'author', 'Make sure we are checking the correct ordering')
+
+        with self.login_user_context(self.get_superuser()):
+            url = self.get_admin_url(model, "changelist")
+            url = f'{url}?o={order + 1}'
+            response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
```

Comment:
This is not testing the ordering, you cannot be sure that the response is in the correct order. Will need ot check that the loaded data is in the right order, may be a little tricky.

Reply:
While you are correct that it is not testing the ordering, it is testing the response of the ordering which is what the issue was. We had a 500 error before and this tests that there is no such error. 

If you would like another tests that test the alphabetical order of authors, I'll try to arrange a test for that as well.